core/org.openjdk.jmc.flightrecorder/src/main/resources/org/openjdk/jmc/flightrecorder/jdk/messages/internal/messages.properties

```diff
@@ -446,4 +446,7 @@ AGGR_VM_OPERATION_DURATION=VM Operation Duration
 AGGR_VM_OPERATION_DURATION_DESC=The sum of the durations for the selected VM operation events.
 AGGR_COMPILATIONS_COUNT=Compilations
 AGGR_COMPILATIONS_COUNT_DESC=The number of compilations
-
+ATTR_SYSTEM_LOAD=System Mode CPU Load
+ATTR_SYSTEM_LOAD_DESC=CPU used by Thread in System mode
+ATTR_USER_LOAD=User Mode CPU Load
+ATTR_USER_LOAD_DESC=CPU used by Thread in User mode
```

Comment:
I think this files misses the `ATTR_JAVA_THREAD` and `ATTR_JAVA_THREAD_DESC`.

Also shouldn't this file be sorted alphabetically ?

Reply:
Thanks for pointing it out. I have added that in recent commit. Regarding alphabetical sorting I have checked the file its like all ATTR_* are grouped together and then it has all AGGR_* so I have added these new values with ATTR_*. I hope that's fine.